assembler/manual.html

```diff
@@ -179,8 +179,8 @@ <h3>2.1 Downloading SPAdes Linux binaries</h3>
         <li><code>hammer</code>  (read error correcting module for Illumina reads)</li>
         <li><code>ionhammer</code>  (read error correcting module for IonTorrent reads)</li>
         <li><code>spades</code>  (assembly module)</li>
-        <li><code>bwa-spades</code>  (<a href="http://bio-bwa.sourceforge.net" target="_blank">BWA</a> alignment module which is required for mismatch correction)</li>
-        <li><code>corrector</code>  (mismatch correction module)</li>
+        <li><code>bwa-spades</code>  (<a href="http://bio-bwa.sourceforge.net" target="_blank">BWA 0.7.12-r1039</a> bundled binary which is required for mismatch correction of final contigs)</li>
```

Comment:
I does not much sense to repeat mentioning versions and dependencies again and again.